gpu4pyscf/dft/tests/test_ao_values.py

```diff
@@ -27,14 +27,14 @@ def setUpModule():
 '''
     mol_sph = pyscf.M(
         atom=atom,
-        basis='ccpvqz',
+        basis='cc-pv6z',
```

Comment:
Adding a custom basis like `basis=('cc-pvqz', [[5, [1.2, 1.]], [6, [1.2, 1.]]])`, instead of using cc-pv6z.